client/src/components/Header/Header.js

```diff
@@ -65,12 +65,12 @@ const ProfileButton = () => {
                 </span>
             </DropdownMenuItemLink>
             <Divider dense />
-            <DropdownMenuItemLink to="/user">Your apps</DropdownMenuItemLink>
+            <DropdownMenuItemLink to="/user">My apps</DropdownMenuItemLink>
             <DropdownMenuItemLink to="/user/organisations">
-                Your organisations
+                My organisations
             </DropdownMenuItemLink>
             <DropdownMenuItemLink to="/user/apikey">
-                Your API keys
+                My API keys
```

Comment:
This is probably fine for now to be consistent with `My Apps`, but interestingly GitHub uses `Your` in their profile menu...

<img width="237" alt="Screen Shot 2021-12-06 at 14 28 00" src="https://user-images.githubusercontent.com/947888/144863698-b7ccffa2-f704-4ae4-84bd-18b32b19f6f8.png">

Reply:
(API Keys probably should be in a general `Settings` page, eventually, but fine for now)

Reply:
Sure, we can defer the renaming. It's not really important, and will probably make our other docs/guides inconsistent.